config/config.go

```diff
@@ -452,6 +475,37 @@ func NewConfig() (c *Config) {
 	return
 }
 
+// Add Health Default Configuration
+func (conf *Config) AddHealthDefault() {
+	// Health default configuration
+	healthConfig := HealthConfig{
+		Rate: []Rate{
+			{
+				Namespace: ".*",
+				Kind:      ".*",
+				Name:      ".*",
+				Tolerance: []Tolerance{
+					{
+						Code:      "^5\\d\\d$",
```

Comment:
I don't remember the previous defaults, if any, but it could be good time to ask folks.
Personally I think we should have some tolerance on the 404, but fail on the rest 4xx and 5xx.

I know this is opinionated but also @jshaughn @jotak have some thoughts on it.

Also, I wouldn't use the "0.1" value, just "0" or empty I think, a "0.1" tend to be read as 10% instead of 0.1% which perhaps has less sense.

Reply:
I don't really know what to say about 404s as a special case of 4xx client errors.  It would be simpler semantics if the default applied to 4xx completely.  Either ignore or set a high tolerance for client errors, maybe like 50%.  If a service is getting killed with client errors it could be a DNS attack or something like that.

As for the 0.1, I'm OK if Degraded is changed to be >0 errors for the time period.

Reply:
@aljesusg @jshaughn I'm ok to apply tolerance for 4xx (client errors).
Perhaps we can set something like starting to degraded in 25% of client errors and show failure state in 50% (as default).
And this is just a starting point, I can imagine when this feature is in place there were more cases.

Reply:
WE must add it again to be align with traffic tab and graph

Reply:
@aljesusg Not sure what you mean, your comment is confusing.
Graph should be synced if 4xx are present or not.
All elements that show health state should read the health config from the same source.

Another discussion is if we consider 4xx error healthy or unhealthy.

Reply:
+1 for more tolerance on 4xx

Reply:
ok, let's tweak the defaults:
- 5xx and grpc errors degraded starting in 0, failure starts on 10%.
- 4xx starts to degraded in 20%, failure on 30% (I know this is pure opinion, just to have a starting point).

Reply:
Agree with that. what do you think @jotak @jshaughn ?

Reply:
I would put something in-between for 4xx, like 10% / 20% ; because, as I said above, for outbound connections this is *really* errors that lies in user's services.

Reply:
(but no problem to do as @lucasponce suggests and eventually tweak that later)

Reply:
so...proposals:

5xx and grpc errors degraded starting in 0, failure starts on 10%.
4xx starts to degraded in 20%, failure on 30%

or

5xx and grpc errors degraded starting in 0, failure starts on 10%.
4xx starts to degraded in 10%, failure on 20%

Reply:
I don't mean to slow down things by commenting. It's fine to use what lucas suggests.

Reply:
we have time while https://github.com/kiali/kiali-ui/pull/1831 is reviewed

Reply:
I'm fine with 5xx 0%/10% because there is some indication of a possible issue as soon as there are any 5xx.   For 4xx I think we need to assume that client errors are potentially bad and so I'm in favor of the lower setting for degraded, 10%.  I'm less concerned with the Failure percentage but to be consistent with a 10% delta above degraded, I'd go with 20%.

Reply:
ok so 10-20. Anyway the user can overwrite this configuration if he wants other values